packages/plugin-packages/mashroom-portal/src/frontend/portal-client/js/MashroomPortalMessageBusImpl.js

```diff
@@ -46,7 +46,8 @@ export default class MashroomPortalMessageBusImpl implements MashroomPortalMaste
             const socketProtocol = (global.location.protocol === 'https:' ? 'wss:' : 'ws:');
             const host = global.document.location.hostname;
             const port = global.document.location.port ? `:${global.document.location.port}` : '';
-            const connectUrl = `${socketProtocol}//${host}${port}${REMOTE_MESSAGING_CONNECT_PATH}`;
+            const clientId = global.sessionStorage.getItem('clientId');
```

Comment:
Please name sessionStorage item unique to avoid collisions with items set by Microfrontends - e.g. MashroomPortalWebSocketClientId - and please put it into a constant